extensions/cross_connect/cross_connect.yml

```diff
@@ -0,0 +1,61 @@
+---
+# yaml-language-server: $schema=https://schema.infrahub.app/infrahub/schema/latest.json
+version: "1.0"
+
+nodes:
+  - name: CrossConnect
+    namespace: Dcim
+    description: Cross-connect between different endpoints within a datacenter.
+    label: Cross-Connect
+    icon: streamline:arrow-crossover-right-solid
+    inherit_from:
+      - DcimConnector
+    order_by:
+      - provider__name__value
+    display_labels:
+      # TODO: Add provider name here when display label supports relationships
+      - identifier__value
+    human_friendly_id:
+      - provider__name__value
+      - identifier__value
+    uniqueness_constraints:
+      - [provider, identifier__value]
+    attributes:
+      - name: identifier
+        kind: Text
+        order_weight: 1000
+      - name: description
+        kind: Text
+        optional: true
+        order_weight: 1300
+      - name: status
+        kind: Dropdown
+        optional: true
+        order_weight: 1200
+        choices:
+          - name: connected
+            label: Connected
+            description: "Fully operational and currently in connected"
+            color: "#7fbf7f"
+          - name: planned
+            label: Planned
+            description: "In the process of being set up"
+            color: "#ffff7f"
+          - name: reserved
+            label: Reserved
+            description: "Fully connected and reserved for a future use"
+            color: "#9090de"
+    relationships:
+      - name: location
+        label: Location
+        peer: LocationHosting
+        optional: false
+        cardinality: one
+        kind: Attribute
+        order_weight: 1100
+      - name: provider
+        peer: OrganizationProvider
+        optional: false
+        cardinality: one
+        kind: Attribute
+        order_weight: 900
```

Comment:
I would find it nicer to have the identifier first on the left before the provider.